### ISSUE TITLE
BrowserSubProcess crashes while playing webm video

### ISSUE THREAD

**Issue by ali-hs** [cefsharp/CefSharp#1520]
My app displays videos in the form of news feed. It has a 5 second preview video for each video which starts playing on load then remain playing in a loop.  After few mins of normal operation, my app crashes with the following log. I am using cefsharp version 43. 

```
Application: CefSharp.BrowserSubprocess.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.NullReferenceException
Stack:
   at CefSharp.Internals.JavascriptCallbackRegistry.Deregister(Int64)
   at <Module>.CefSharp.CefAppUnmanagedWrapper.OnProcessMessageReceived(CefSharp.CefAppUnmanagedWrapper*, CefRefPtr<CefBrowser>*, cef_process_id_t, CefRefPtr<CefProcessMessage>*)
   at <Module>.CefExecuteProcess(CefMainArgs*, CefRefPtr<CefApp>*, Void*)
   at <Module>.CefExecuteProcess(CefMainArgs*, CefRefPtr<CefApp>*, Void*)
   at CefSharp.CefAppWrapper.Run()
   at CefSharp.BrowserSubprocess.Program.Main(System.String[])
```

**Comment by amaitland** [comment]
> I am using cefsharp version 43.

Specifically `43.0.0` or `43.0.1`? If you haven't upgraded, it's probably wise to as there were a lot of bugs fixed.

Also does the problem reproduce in `45.0.0`? If it does then can you provide a demo to reproduce the problem. For convenience you can fork the `MinimalExample`. (No zip files please)

https://github.com/cefsharp/CefSharp.MinimalExample

**Comment by amaitland** [comment]
@ali-hs Anything more on this?

**Comment by ali-hs** [comment]
@amaitland Sorry for the late reply, I have upgraded to version `45.0.0` and tested it on different windows systems for couple of weeks. Seems like this issue is resolved in `45.0.0`
